### PR TITLE
Convert to ES6 modules

### PR DESCRIPTION
**Pull request by kaicataldo**

### CODE REVIEW

File: src/Rule.js [hunk 1]
```diff
@@ -1,84 +1,80 @@
-define([
-  'nbd/Class',
-  'nbd/util/extend',
-  'nbd/util/diff',
-  '../util/css',
```
Comment (nemtsov) [inline src/Rule.js]: missing `../util/css`

Comment (kaicataldo) [inline src/Rule.js]: Great catch - meant to leave a note - this actually isn't used anywhere in the file. Someone probably just forgot to remove it!

Comment (nemtsov) [inline src/Rule.js]: ah, cool

**Comment by nemtsov** [comment]
👍

**Comment by agaripian** [comment]
👍

**Pull request merged**